### PR TITLE
Unused import (F401) reported at the bottom of the file

### PR DESCRIPTION
**Pull request by hvdklauw**
I see in the code there is some special handling in there, but it's unclear to me why it would report the issue at the bottom of the file.

### CODE REVIEW

**Comment by hvdklauw** [comment]
Glad hub's issue to pull request conversion still works.

It wasn't clear to me on how to run the tests, npm test didn't work.

If you need tests added let me know and I'll have a look.

File: lib/main.coffee [hunk 1]
```diff
@@ -1,4 +1,5 @@
-tokenizedLineForRow = (textEditor, lineNumber) -> textEditor.displayBuffer.tokenizedBuffer.tokenizedLineForRow(lineNumber)
+tokenizedLineForRow = (textEditor, lineNumber) ->
+  textEditor.displayBuffer.tokenizedBuffer.tokenizedLineForRow(lineNumber)
```
Comment (Arcanemagus) [inline lib/main.coffee]: :+1:

**Comment by Arcanemagus** [comment]
Test for this would be nice, but as there is already a ton of other things that still need specs I'm fine with merging this as is, thanks!

And what is this magic Issue to Pull Request conversion?!

**Comment by hvdklauw** [comment]
Hub from github itself has a command to create a pull request and you can
pass it a issue number to use.

It will raise a warning saying it might be deprecated but as long as it
works it saves a lot of management between pull requests and issues it
fixes.

**Comment by Arcanemagus** [comment]
Published in v1.10.1, and thanks for the info!

**Pull request merged**